nntrainer/tensor/blas_neon.h

```diff
@@ -65,13 +65,29 @@ void copy_int4_to_fp32(const unsigned int N, const uint8_t *X, float *Y);
  */
 void copy_int8_to_fp32(const unsigned int N, const uint8_t *X, float *Y);
 
+/**
+ * @brief     copy function with neon: Y = X
+ * @param[in] N number of elements in X
+ * @param[in] X int8_t * for Vector X
+ * @param[in] Y float * for Vector Y
+ */
+void copy_int8_to_fp32(const unsigned int N, const int8_t *X, float *Y);
+
 /**
  * @brief     copy function with neon: Y = X
  * @param[in] N number of elements in X
  * @param[in] X uint8_t * for Vector X
  * @param[in] Y uint8_t * for Vector Y
  */
 void copy_int8_or_int4(const unsigned int N, const uint8_t *X, uint8_t *Y);
+
+/**
+ * @brief     copy function with neon: Y = X
+ * @param[in] N number of elements in X
+ * @param[in] X int8_t * for Vector X
+ * @param[in] Y int8_t * for Vector Y
+ */
+void copy_int8(const unsigned int N, const int8_t *X, int8_t *Y);
```

Comment:
I wasn't aware of signed/unsigned int8 case when implementing copy_int8_or_int4..
How about using copy_s8 / copy_u8 for better understanding?

Reply:
copy_s8 and copy_u8 make more sense :) Let's create a new PR to rename the functions!